### ISSUE TITLE
Play a sound for Granite.MessageDialog

### ISSUE THREAD

**Issue by hanaral** [elementary/granite#471]
## Problem

Many users may react slowly or not be able to immediately notice an error popup. The user has to be staring at their to notice a popup, which is pretty bad if it is a critical error.

## Proposal

Gala should play a crisp 'dunn' sound effect when a Granite.MessageDialogue with an error appears, and for when a Granite error views appears. 
This should be done on an opt-in basis to avoid accidentally confusing users.
I'd also suggest adding a lighter sound effect for authentication popups.

tl;dr
Error views and granite error dialogues should have unique audio cues 

## Prior Art

_Maybe I'll add some later_

**Comment by danrabbit** [comment]
Since this would need context, it would have to happen in Granite. The Window Manager isn't aware of the dialog's context, just that it's a dialog.

This would be hard to enforce consistently since apps may choose not to use Granite.MessageDialog to show errors

**Comment by cassidyjames** [comment]
I’m pretty :-1: about dialogs playing sounds in general; notifications are the platform way to notify for background tasks, and dialogs generally assume the user is present. I believe GNOME used to (optionally) play an FD.o sound when dialogs were thrown, but it was removed because it was annoying. It's probably worthwhile to investigate when and why they removed it if you want to petition for it to be re-added.

**Comment by hanaral** [comment]
This isn't about notifying the user about a backrgound activity, you've already pointed out that the general way to notify is through notifications. 
This is supposed to be a way for there to automatically be an auditory cue to fit with something of importance on screen - yes, I could use a screen reader, but what happens to the using accessibility features as features?
I think that this should just be something the user can disable like window animations.

Also is there no way for granite.messagedialog to emit a dbus signal or something that gala or even the notification server(which already plays sounds) can catch?
